web/src/types.ts

```diff
@@ -6,13 +6,67 @@ export type MatchedMessage = {
 
 // TODO/typescript: Move this to message_store
 export type MessageType = "private" | "stream";
+export type MessageContentType = "text/html" | "text/x-markdown";
+export type MessageReactionType = "unicode_emoji" | "realm_emoji" | "zulip_extra_emoji";
+
+// TODO/typescript: Move these types to message_store
+
+export type DisplayRecipient = {
+    email: string;
+    full_name: string;
+    id: number;
+    is_mirror_dummy: boolean;
+};
+
+export type MessageEditHistory = {
+    user_id: number | null;
+    timestamp: number;
+    prev_content: string;
+    prev_rendered_content_version: number;
+    prev_stream: number;
+    prev_topic: string;
+    stream: number;
+    topic: string;
+};
```

Comment:
Several of these fields might not appear, right? Or am I remembering wrong?

Reply:
Yup my bad misread it on the docs, fixed it!